cahiers-d-ethnomusicologie.csl

```diff
@@ -0,0 +1,484 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="fr-FR">
+  <info>
+    <title>Cahiers d'ethnomusicologie (French)</title>
+    <id>http://www.zotero.org/styles/cahiers-d-ethnomusicologie</id>
+    <issn>1662-372X</issn>
+    <eissn>2235-7688</eissn>
+    <link
+	href="http://www.zotero.org/styles/cahiers-d-ethnomusicologie"
+	rel="self"/>
+    <link href="http://www.zotero.org/styles/iso690-author-date-en"
+	  rel="template" />
+    <link
+	href="http://www.adem-geneve.com/fr/cahiers-d-ethnomusicologie-contribution"
+	rel="documentation" />
+    <author>
+      <name>Victor A. Stoichita</name>
+      <email>svictor@svictor.net</email>
+      <uri>http://www.svictor.net</uri>
+    </author>
+    <category citation-format="author-date"/>
+    <summary>Style based on ISO 690:2010(F), V2 adapted from a template by L.
+    Mellifluo, R. Grolimund, M. Hardegger and P.-A. Giraud.</summary>
+    <updated>2016-09-24T23:32:13+00:00</updated>
+    <rights
+    license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="anonymous">Anon.</term>
+      <term name="no date">[sans date]</term>
+      <term name="in">in</term>
+      <term name="online">en ligne</term>
+      <term name="accessed">consulté le</term>
+      <term name="retrieved">disponible</term>
+      <term name="from">à l'adresse</term>
+      <term name="editor" form="short">
+	<single>dir</single>
+	<multiple>dirs.</multiple>
+      </term>
+      <term name="page" form="short">
+        <single>p.</single>
+        <multiple>p.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" name-as-sort-order="first" initialize="false" sort-separator=" " delimiter=", " delimiter-precedes-last="never">
+        <name-part name="family" text-case="uppercase"/>
+        <name-part name="given"/>
+      </name>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" name-as-sort-order="all" form="short" sort-separator=", " delimiter=", " delimiter-precedes-last="never">
+        <name-part name="family"/>
+        <name-part name="given"/>
+      </name>
+      <label prefix=" (" form="short" suffix=".)"/>
+    </names>
+  </macro>
+  <macro name="translator">
+    <names variable="translator">
+      <name and="text" name-as-sort-order="all" sort-separator=", " delimiter=", " delimiter-precedes-last="never">
+        <name-part name="family" text-case="uppercase"/>
+        <name-part name="given"/>
+      </name>
+      <label prefix=" (" form="short" suffix=".)"/>
+    </names>
+  </macro>
+  <macro name="responsability">
```

Comment:
I'm pretty flexible with macro names, but I don't think we should use a non-existent English word. Either fix this (and all calls to it) to `responsibility` or maybe use the `responsable` which works in French and English (I think?)